build.py

```diff
@@ -310,7 +312,7 @@ def qtmake(*args):
         qt_configure_args.append('-mp')
 
     # Build Qt 4 with clang on OS X
-    if sys.platform == 'darwin' and os.path.isfile('/usr/bin/clang') and not is_qt5():
+    if sys.platform == 'darwin' and os.path.isfile('/usr/bin/clang') and os.path.isfile('/usr/bin/clang++') and not is_qt5():
```

Comment:
Can you split this conditional over multiple lines to make it more readable? Something like:

```python
if cond1 \
        and cond2 \
        and not cond3:
    # body
    # ...
```

Should do the trick.